src/cmd/operator/csi_driver.go

```diff
@@ -95,7 +96,8 @@ func setupCSIDriver(ns string, cfg *rest.Config) (manager.Manager, func(), error
 		return nil, cleanUp, err
 	}
 
-	if err := csiprovisioner.NewOneAgentProvisioner(mgr, csiOpts, access).SetupWithManager(mgr); err != nil {
+	maxParallelDownloads, _ := strconv.ParseInt(os.Getenv("MAX_PARALLEL_DOWNLOADS"), 10, 64)
```

Comment:
Error handling is necessary here (testcase: `MAX_PARALLEL_DOWNLOADS=`:trollface:)

Reply:
Alongside the error handling I would appreciate sane limits, i.e., min and max values, to handle cases like "-1".
All of this could be handled in a function which reads the env var, parses it into an int, checks the value, enforces the limits and then returns the final value